File: CodingBat/Interpret/interpret.java
// https://codingbat.com/prob/p294185

public int interpret(int value, String[] commands, int[] args) {
  for (int i = 0; i < commands.length; i++) {
    if (commands[i] == "+") {
      value += args[i];
    }
    else if (commands[i] == "-") {
      value -= args[i];
    }
    else if (commands[i] == "*") {
      value *= args[i];
    }
    else {
      return -1;
    }
  }
  
  return value;
}

File: CodingBat/Interpret/interpret.py
// https://codingbat.com/prob/p234011

def interpret(value, commands, args):
  for command, arg in zip(commands, args):
    if (command == "+"):
      value += arg
    elif (command == "-"):
      value -= arg
    elif (command == "*"):
      value *= arg
    else:
      return -1
  
  return value
